ecs-init/exec/iptables/iptables.go

```diff
@@ -189,18 +203,52 @@ func getBlockIntrospectionOffhostAccessInputChainArgs() []string {
 	return []string{
 		"INPUT",
 		"-p", "tcp",
-		"-i", getOffhostIntrospectionInterface(),
+		"-i", defaultOffhostIntrospectionInterface,
 		"--dport", agentIntrospectionServerPort,
 		"-j", "DROP",
 	}
 }
 
-func getOffhostIntrospectionInterface() string {
+func getOffhostIntrospectionInterface() (string, error) {
```

Comment:
i think the documentation for ECS_OFFHOST_INTROSPECTION_INTERFACE_NAME env in README should be updated as part of this change

Reply:
You are right, which reminds me we will also need to update the public Agent configuration [doc](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html) when the change is out.